app/views/spaces/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <% breadcrumb :edit_space, space %>
 
-<% if space.entrance_id.blank? || space.rooms.blank? || space.members.blank? %>
+<% if space.rooms.blank? || space.entrance_id.blank? || space.members.blank? || space.entrance&.furnitures.blank? %>
```

Comment:
Could we pull this up to `Space#configured?` so that the view only has one method to call?

Reply:
added in 7f31d6aa0264553d84944a07f2f9f7f4beddf8f8 but super janky and at least the page loads 😅 

Tho I added it to Space model. I can't recall if views invoking model methods was a correct rails or not. 🤦 

I tried adding it to the Space controller as a helper method or regular method.... but it broke the page. I'll keep trying. 

I might be overthinking this.... 😅